cql3/statements/alter_index_statement.cc

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) 2025-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+ */
+
+#include "cql3/statements/alter_index_statement.hh"
+
+#include "cql3/query_processor.hh"
+#include "cql3/statements/cf_prop_defs.hh"
+#include "cql3/statements/view_prop_defs.hh"
+#include "data_dictionary/data_dictionary.hh"
+#include "exceptions/exceptions.hh"
+#include "index/secondary_index_manager.hh"
+#include "replica/database.hh"
+#include "service/migration_manager.hh"
+#include "validation.hh"
+
+namespace cql3::statements {
+
+namespace {
+
+sstring get_view_name(const sstring& index_name) {
```

Comment:
Nitpick: for me, "static" is much clearer than an anonymous namespace. You have a clear indication on each static function that it is file-local. What is the benefit of an anonymous namespace?

Reply:
Ok. I won't insist. Resolving the conversation.